### PR TITLE
[Windows Upgrade] Hotfix: added backward compatibility of new DC/OS Linux installer with upgrade func vs old Ansible Windows BETA container

### PR DESCRIPTION
**Pull request by sergiimatusEPAM**
## High-level description

There is issue in broken compatibility of old Ansible container "mesosphere/dcos-ansible-bundle:windows-beta-support" with recent (new) changes to DC/OS installer(s) - another word move of dcos_install.ps1 from DC/OS Windows installer to DC/OS Linux installer.
There is a task under Bootstrap role called "Unpack Windows installer packages" in Ansible. How does it work - if dcos_install.ps1 missing on file system, then Ansible unpacks Windows installer. Otherwise skip the task. Another word - the dcos_install.ps1 used as trigger to run the task. And now this file is unpacked earlier by DC/OS Linux installer.

To resolve the issue unpack of dcos_install.ps1 by DC/OS Windows Installer and DC/OS Linux Installer proposed to  different locations:
Windows' one -  to windows/prerequisites/dcos_install.ps1 
Linux's one - to windows/dcos_install.ps1 

Therefore if user has old Windows BETA Ansible container, Windows's one used. If Windows GA Ansible container used - a Linux's one used and it 

## Corresponding DC/OS tickets (required)

  - [D2IQ-61891](https://jira.mesosphere.com/browse/D2IQ-61891) EPIC : Support Windows Node Update

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot changelog-not-required Windows Upgrade minor func

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review  @jeschkies

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review  @jongiddy

**Comment by sergiimatusEPAM** [comment]
@mesosphere-mergebot request-review  @DominikDary

File: gen/build_deploy/powershell/dcos_install.ps1 [hunk 1]
```diff
@@ -0,0 +1,422 @@
+<#
```
Comment (janisz) [inline gen/build_deploy/powershell/dcos_install.ps1]: can we delete [gen/build_deploy/powershell/dcos_install.ps1.in](gen/build_deploy/powershell/dcos_install.ps1.in) then?

**Comment by sergiimatusEPAM** [comment]
No, it will be used by upgrade part. The format changes there

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: gen/build_deploy/powershell/dcos_install.ps1 (hunk 1 above)
Comment (janisz) [reply gen/build_deploy/powershell/dcos_install.ps1]: There are only 4 lines different. I prefer to not copy whole file just to change mandatory parameter setting.

```diff
# diff gen/build_deploy/powershell/dcos_install.ps1{,.in}                                          
45a46,48
> # Metadata:
> #   dcos image commit : {{ dcos_image_commit }}
> #   generation date   : {{ generation_date }}
51,52c54,55
<     [Parameter(Mandatory=$true)] [string] $bootstrap_url,
<     [Parameter(Mandatory=$true)] [string] $masters,
---
>     [Parameter(Mandatory=$false)] [string] $bootstrap_url = '{{ bootstrap_url }}',
>     [Parameter(Mandatory=$false)] [string] $masters = '{{ master_list }}',
```

**Comment by sergiimatusEPAM** [comment]
> @sergiimatusEPAM Please add proper description what's broken and why this changes are fixing it

Added description

File: gen/build_deploy/powershell/dcos_install.ps1 (hunk 1 above)
Comment (sergiimatusEPAM) [reply gen/build_deploy/powershell/dcos_install.ps1]: Within GA scope dcos_install.ps1 was replaced in favor of template. Extra steps were done to make this filled up in Python. Now, to full fill your request it needs to be done in Powershell - use a new template, but make a substitution within a build_genconf_windows.ps1 . 
However this work will be redundant in GA

**Comment by sergiimatusEPAM** [comment]
@janisz , as requested extra dcos_install.ps1 was removed permanently

**Comment by DominikDary** [comment]
@mesosphere-mergebot changelog-not-required internal change

**Pull request merged**